src/System.IO.Pipelines/System.IO.Pipelines.csproj

```diff
@@ -2,7 +2,7 @@
   <Import Project="..\..\tools\common.props" />
   <PropertyGroup>
     <Description>An abstraction for doing efficient asynchronous IO</Description>
-    <TargetFramework>netstandard1.1</TargetFramework>
+    <TargetFrameworks>netstandard1.1;netstandard2.0;netcoreapp2.1</TargetFrameworks>
```

Comment:
This is probably the most unfortunate part of this change but meh..

Reply:
Meh indeed. It's in implementation details with zero bearings on the user's experience.

Reply:
> It's in implementation details with zero bearings on the user's experience.

In this case yes.